CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# Thank You
+
+We are grateful that you considered contributing to metaKGP-wiki. Please follow the below guidelines for contribution.
+
+Follow the golder rule
+
+> Code, Document, Raise Issues and Review like you have to explain everything to a new developer without saying a word.
+
+## First Time
+
+The first thing to do always is to run the program and note how it works. One of the best ways to do this is to launch the application and go through the files that it launches in the sequential manner.
+
+Also make note of the coding style and dependencies that the project follows.
+
+## Choosing issues
+
+It is recommended to go through the issues tab after you have run the program once and look for easy issues.
+
+If an issue is not assigned; follow through the comments, ask queries about it and then start working on it.
+
+## Creating issues
+
+If you face any reproducable problem while launching the docker containers or any feature you think should be there, file an issue in the github repository.
```

Comment:
> If you face any reproducable problem while launching the docker containers or any feature you think should be there

I don't know what you are trying to say here. Is it "If you face a reproducable problem while launching the docker containers, please report an issue here. If you think there is something missing in our documentation, please open a pull request adding the documentation."?

Reply:
I've changed it to the suggested sentence :)